tests/unit/EntityInfo.spec.ts

```diff
@@ -157,3 +159,31 @@ describe('Entity Info component in a Correction as a numbered company', () => {
     expect(wrapper.find('#entity-business-phone').text()).toBe('321-456-7890')
   })
 })
+
+describe('Entity Info component for a firm', () => {
+  it('displays operating name correctly for a SP Change filing', () => {
+    const wrapper = shallowWrapperFactory(
+      EntityInfo,
+      null,
+      {
+        entitySnapshot: {
+          businessInfo: {
+            alternateNames: [
+              { identifier: 'FM1234567', operatingName: 'My Operating Name' }
+            ],
+            legalName: 'My Legal Name'
+          }
+        },
+        tombstone: {
+          businessId: 'FM1234567',
+          entityType: CorpTypeCd.SOLE_PROP,
+          filingType: FilingTypes.CHANGE_OF_COMPANY_INFO
+        }
+      }
+    )
+
+    expect(wrapper.find('#entity-legal-name').text()).toBe('My Operating Name')
+
+    wrapper.destroy()
+  })
```

Comment:
I didn't add unit tests for firm (SP) conversion and correction filings, because the code to compute Legal Name/ Operating Name and display it in the Entity Info header is the same for all 3 filings.